### DISCUSSION TITLE
Edit events

### DISCUSSION THREAD

**Discussion by JonnyMillar29** [jynnie/soullocke#3] [Ideas]
Would it be possible to add an edit button to this screen - if I've accidentally entered the wrong route for when a pokemon is moved to the grave then I can't find a way to fix that beyond deleting the encounter and re-entering the details. Not a massive pain if you're only tracking deaths like me, but if someone is using the tool to its full capability (tracking box or daycare moves) then it could be very annoying to have to redo everything.

Thanks :)
![image](https://user-images.githubusercontent.com/30133073/170025913-9aae9b51-dad8-4623-b449-3c10d64e0bbf.png)

**Comment by jynnie** [comment]
Sorry for the late response 😅 Totally!

**Comment by jynnie** [comment]
Editing & deleting events has been added! Here's a little gif of how it works. If you discover any issues with it, let me know.

![2022-09-08 00 03 40](https://user-images.githubusercontent.com/7284723/189057300-9c8da9aa-4635-45cb-9aa2-43d96c7cad9f.gif)
